### ISSUE TITLE
What is the minimum hardware requirement to fine-tune I3D on new dataset?

### ISSUE THREAD

**Issue by stephencheung16** [deepmind/kinetics-i3d#43]
Dear Sir/Madam,

Can the pre-trained model run in a PC of i7 and GTX1080 for fine-tuning?  The size of the new dataset will be about 100.

Thanks.

Regards,
Stephen

**Comment by salmedina** [comment]
Yes, that is enough, should take less than two days.

**Comment by stephencheung16** [comment]
Thanks.

**Comment by tadbeer** [comment]
Hi @salmedina and @stephencheung16 . I am trying to replicate the experiment of finetuning i3d trained on Kinetics, for ucf-101, as per the methods specified in the paper, but getting unsuccessful in achieving any significant accuracy (not even on training data) even after several iterations. Could you please give any guidance?
To simplify, I just used 10 classes from ucf-101,
extracted 64 contiguous frames from each video, from any (feasible) random begin point for each epoch,
spatially cropped 224x224 from video resized to 256,
used Adam optimiser with initial learning rate(1e-4),
and mean subtarction and division by 255 from data.
